src/types/modules.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "emoji-from-text" {
+  function emojiFromText(text: string, onlyOneMatch?: boolean): any;
+  export = emojiFromText;
+}
```

Comment:
There is no `@types/emoji-from-text` package so I had to google this syntax a bit